File: Codigos estudiantes por lenguaje/Java/Cristian Maldonado/Geometria/Arc_length.java
public class Arc_Length {

    static double arcLength(double diameter, double angle) {
        double pi = 22.0 / 7.0;
        double arc;
        if (angle >= 360) {
            System.out.println("Angulo no puede ser formado");
            return 0;
        } else {
            arc = (pi * diameter) * (angle / 360.0);
            return arc;
        }
    }

    public static void main(String[] args) {
        double diameter = 25.0;
        double angle = 45.0;
        double arc_len = arcLength(diameter, angle);
        System.out.println(arc_len);
    }
}

File: Codigos estudiantes por lenguaje/C++/Cristian Maldonado/Geometria/Arc_length.cpp
#include <iostream>
#include <iomanip>

using namespace std;

typedef long double ld;

ld arcLength(ld diametro, ld angulo){
    ld pi = 22.0/7.0;
    ld arc;
    if(angulo >= 360){
        cout<<"Angulo no puede ser formado"<<endl;
        return 0;
    }else{
        arc = (pi * diametro) * (angulo / 360.0);
        return arc;
    }
}

int main()
{
    //Inicialmente manejado en grados
    ios_base::sync_with_stdio(false);cin.tie(NULL);cout.tie(NULL);
    cout.precision(22);
    ld diametro, angulo; cin>>diametro>>angulo;
    cout<<arcLength(diametro, angulo);
}
